app/services/affiliate-slots.js

```diff
@@ -77,6 +77,35 @@ const checkMobileSystem = (unit) => {
   return true;
 };
 
+
+/**
+ * Convert service response to flat structure
+ *
+ * @param {Object} response
+ * @returns {Targeting[]}
+ */
+const flattenKnowledgeGraphTargeting = (response) => {
+  // convert API to nicer, more useful format
+  const targeting = [];
+
+  // convert from tree structure to flat structure for easier comparison later
+  response.forEach((e) => {
```

Comment:
Can you use more descriptive variables than single letters?

Reply:
d3f3720